frameworks/python/django/Dockerfile

```diff
@@ -4,9 +4,8 @@ RUN apk add --no-cache mariadb-connector-c
 
 ADD src/requirements.txt /src/requirements.txt
 RUN \
-	apk add --no-cache --virtual .build-deps gcc libc-dev mariadb-dev && \
-	pip install --no-cache-dir -r /src/requirements.txt && \
-	apk del .build-deps
+	apk add --no-cache --virtual .build-deps gcc libc-dev mariadb-dev zlib-dev jpeg-dev gcc musl-dev && \
```

Comment:
If some of these are required at runtime (e.g. if you need `zlib` and `jpeg`, for example), let's install them in a separate step and not keep the build dependencies installed afterward...something like this:
```dockerfile
RUN apk add --no-cache zlib jpeg
ADD src/requirements.txt /src/requirements.txt
RUN \
    apk add --no-cache --virtual .build-deps gcc libc-dev mariadb-dev zlib-dev jpeg-dev gcc musl-dev && \
    pip install --no-cache-dir -r /src/requirements.txt && \
    apk del .build-deps
```

This will keep the image size down and maximize layer reuse for iteration :)